torch/linalg/__init__.py

```diff
@@ -234,7 +234,7 @@
     Consider using :func:`torch.linalg.solve` if possible for multiplying a matrix on the left by
```

Comment:
This may probably better written as ``:func:`~linalg.solve` `` to match the reference below?

Reply:
:func:`~linalg.solve` doesn't hyperlink to the docs though so I think it preferable to leave it as :func:`torch.linalg.solve`